controllers/admin/AdminCarriersController.php

```diff
@@ -649,6 +649,31 @@ public function getList($id_lang, $order_by = null, $order_way = null, $start =
                 $this->_list[$key]['name'] = Carrier::getCarrierNameFromShopName();
             }
         }
+
+        // Add specific sort for naming
+        if ($this->_orderBy == 'name') {
+            usort($this->_list, [$this, 'sortListByName']);
+        }
+    }
+
+    /**
+     * @param array $a
+     * @param array $b
+     *
+     * @return int
+     */
```

Comment:
```suggestion
    /**
     * Sort arrays based on the name
      *
     * @param array $a The first array to compare
     * @param array $b The second array to compare 
     *
     * @return int 0 if they are equal, -1 or 1 if greater depending on the sort order way
     */
```

Could be more explicit ^^

Reply:
Fixed